components/service/glean/src/main/java/mozilla/components/service/glean/error/ErrorRecording.kt

```diff
@@ -41,12 +41,15 @@ object ErrorRecording {
      *     ping.  It does not need to include the metric name, as that is automatically
      *     prepended to the message.
      * @param logger The [Logger] instance to display the warning.
+     * @param numErrors The optional number of errors to report for this [ErrorType].
      */
+    @Suppress("LongMethod")
```

Comment:
Wow, adding that little bit pushed it over the `LongMethod` limit?

Reply:
Yes :| That was a bit surprising to me too, but... meh :|